src/ngraph/op/batch_norm.hpp

```diff
@@ -26,39 +27,40 @@ namespace ngraph
+            /// \param gamma gamma scaling for normalized value. [C]
+            /// \param beta bias added to the scaled normalized value [C]
+            /// \param epsilon Avoids divsion by 0 if input has 0 variance
             BatchNormTraining(std::shared_ptr<Node> input,
                               std::shared_ptr<Node> gamma,
                               std::shared_ptr<Node> beta,
                               double epsilon);
 
-            // \deprecated
-            // In this version of BatchNorm:
-            //
-            // MEAN AND VARIANCE: computed directly from the content of 'input'.
-            //
-            // OUTPUT VALUE: A tuple with the following structure:
-            //   [0] - The normalization of 'input'.
-            //   [1] - The per-channel means of (pre-normalized) 'input'.
-            //   [2] - The per-channel variances of (pre-normalized) 'input'.
-            //
-            // AUTODIFF SUPPORT: yes: 'generate_adjoints(...)' works as expected.
-            //
-            // SHAPE DETAILS:
-            //   gamma:     must have rank 1, with the same span as input's channel axis.
-            //   beta:      must have rank 1, with the same span as input's channel axis.
-            //   input:     must have rank >= 2.  The second dimension represents the channel axis
-            //              and must have a span of at least 1.
-            //   output[0]: shall have the same shape as 'input'.
-            //   output[1]: shall have rank 1, with the same span as input's channel axis.
-            //   output[2]: shall have rank 1, with the same span as input's channel axis.
+            NGRAPH_DEPRECATED_DOC
+            /// In this version of BatchNorm:
+            ///
+            /// MEAN AND VARIANCE: computed directly from the content of 'input'.
+            ///
+            /// OUTPUT VALUE: A tuple with the following structure:
+            ///   [0] - The normalization of 'input'.
+            ///   [1] - The per-channel means of (pre-normalized) 'input'.
+            ///   [2] - The per-channel variances of (pre-normalized) 'input'.
+            ///
+            /// AUTODIFF SUPPORT: yes: 'generate_adjoints(...)' works as expected.
+            ///
+            /// SHAPE DETAILS:
+            ///   gamma:     must have rank 1, with the same span as input's channel axis.
+            ///   beta:      must have rank 1, with the same span as input's channel axis.
+            ///   input:     must have rank >= 2.  The second dimension represents the channel axis
+            ///              and must have a span of at least 1.
+            ///   output[0]: shall have the same shape as 'input'.
+            ///   output[1]: shall have rank 1, with the same span as input's channel axis.
+            ///   output[2]: shall have rank 1, with the same span as input's channel axis.
+            NGRAPH_DEPRECATED("Use another constructor")
```

Comment:
This is a bit vague... maybe `Use BatchNormTraining::BatchNormTraining(input, gamma, beta, epsilon)